src/Monolog/Registry.php

```diff
@@ -96,7 +125,7 @@ public static function removeLogger($logger)
     }
 
     /**
-     * Clears the registry
+     * Clears the registry of loggers
      */
     public static function clear()
```

Comment:
If we decide to use the `$options` array, shouldn't we clear the options here as well?